samples/MvcSample.Web/Models/Person.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace MvcSample.Web
+{
+    public class Person
+    {
+        [Required]
+        public string Name { get; set; }
+
+        [Required]
+        public Address Address { get; set; }
+
+        [Required]
```

Comment:
`MinLength` works only on `string`s and `array`s. Let me validate the `Count` in the `Action`.